src/hevm/src/EVM/UnitTest.hs

```diff
@@ -108,38 +109,31 @@ type ABIMethod = Text
 initializeUnitTest :: UnitTestOptions -> Stepper ()
 initializeUnitTest UnitTestOptions { .. } = do
 
-  -- Maybe modify the initial VM, e.g. to load library code
-  Stepper.evm (modify vmModifier)
+  let addr = testAddress testParams
 
-  -- Make a trace entry for running the constructor
-  Stepper.evm (pushTrace (EntryTrace "constructor"))
+  Stepper.evm $ do
+    -- Maybe modify the initial VM, e.g. to load library code
+    modify vmModifier
+    -- Make a trace entry for running the constructor
+    pushTrace (EntryTrace "constructor")
 
   -- Constructor is loaded; run until it returns code
-  bytes <- Stepper.execFullyOrFail
-  case bytes of
-    SymbolicBuffer bs -> error "unexpected symbolic contract bytecode"
-    ConcreteBuffer bs -> do
-      addr <- Stepper.evm (use (state . contract))
+  void Stepper.execFully
 
-      -- Mutate the current contract to use the new code
-      Stepper.evm $ replaceCodeOfSelf (RuntimeCode bs)
```

Comment:
how come we can omit this now?

Reply:
The `initialUnitTestVm` is now tx for which `create` is true, so finalization of that tx deposits the bytecode